src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: getMaxWidth(
+    $cell-size,
+    $columns-number,
+    $cell-gap,
+    $padding-size
+  );
+  gap: $cell-gap;
+  padding: $padding-size;
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #eee;
+    border: 1px solid #000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffc0ca;
+      transform: translateY(-20px);
+      transition: transform $transition-time, background-color $transition-time;
```

Comment:
move `transition: transform $transition-time, background-color $transition-time;` from hover state to static one (&__cell)